src/main/java/app/sami/languageWeb/api/auth/UserController.java

```diff
@@ -0,0 +1,50 @@
+package app.sami.languageWeb.api.auth;
+
+import app.sami.languageWeb.api.dto.UserDto;
+import app.sami.languageWeb.models.User;
+import app.sami.languageWeb.repos.UserRepository;
+import lombok.AllArgsConstructor;
+import org.apache.coyote.Response;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@RestController
+@RequestMapping("/user")
+public class UserController {
+    @Autowired
+    private UserRepository userRepository;
+
+    @GetMapping("/users")
+    public ResponseEntity<List<UserDto>> getUsers(@RequestParam(defaultValue = "0") int page,
```

Comment:
Again, let's drop `ResponseEntity`. Usually, you'd want a `UsersDto` to return a dictionary instead of returning a `list`.